recipes/SOCI/meta.yaml

```
@@ -0,0 +1,120 @@
+  sha256: fa69347b1a1ef74450c0382b665a67bd6777cc7005bbe09726479625bcf1e29c
+
+build:
+  number: 0
+
+outputs:
+  - name: soci-core
+    script: build_soci.sh  # [unix]
+    script: bld_soci.bat   # [win]
+    requirements:
+      build:
+        - {{ compiler('cxx') }}
+        - cmake
+        - make  # [unix]
+        - ninja  # [win]
+    test:
+      commands:
+        - test -f ${PREFIX}/lib/libsoci_core.so  # [unix]
+        # - if exist %LIBRARY_PREFIX%\lib\libsoci_core.dll (exit 0) else (exit 1)  # [win]
+
+    about:
+      summary: 'SOCI is a general purpose database written in C++.'
+      doc_source_url:  http://soci.sourceforge.net/doc/release/4.0/
+
+  - name: soci-sqlite
+    script: build_soci.sh  # [unix]
+    script: bld_soci.bat   # [win]
+    requirements:
+      build:
+        - {{ compiler('cxx') }}
+        - cmake
+        - make  # [unix]
+        - ninja  # [win]
+      host:
+        - soci-core
+        - sqlite
+      run:
+        - soci-core
+        - sqlite
+
+    test:
+      commands:
+        - test -f ${PREFIX}/lib/libsoci_sqlite3.so  # [unix]
+        # - if exist %LIBRARY_PREFIX%\lib\libsoci_core.dll (exit 0) else (exit 1)  # [win]
+
+    about:
+      summary: 'soci-sqlite3 contains SOCI core and the C++ implementation of SQLite3.'
+      doc_source_url:  http://soci.sourceforge.net/doc/release/4.0/
+
+  - name: soci-mysql
+    script: build_soci.sh  # [unix]
+    script: bld_soci.bat   # [win]
+    requirements:
+      build:
+        - {{ compiler('cxx') }}
+        - cmake
+        - make  # [unix]
+        - ninja  # [win]
+      host:
+        - soci-core
+        - mysql-devel
+      run:
+        - soci-core
+        - mysql
+
+    test:
+      commands:
+        - test -f ${PREFIX}/lib/libsoci_mysql.so  # [unix]
+        # - if exist %LIBRARY_PREFIX%\lib\libsoci_core.dll (exit 0) else (exit 1)  # [win]
+
+    about:
+      summary: 'soci-mysql contains SOCI core and the C++ implementation of MySQL.'
+      doc_source_url:  http://soci.sourceforge.net/doc/release/4.0/
+
+  - name: soci-postgresql
+    script: build_soci.sh  # [unix]
+    script: bld_soci.bat   # [win]
+    requirements:
+      build:
+        - {{ compiler('cxx') }}
+        - cmake
+        - make  # [unix]
+        - ninja  # [win]
+      host:
+        - soci-core
+        - libpq
+      run:
+        - soci-core
+        - libpq
+    test:
+      commands:
+        - test -f ${PREFIX}/lib/libsoci_postgresql.so  # [unix]
+        # - if exist %LIBRARY_PREFIX%\lib\libsoci_core.dll (exit 0) else (exit 1)  # [win]
+
+    about:
+      summary: 'soci-postgresql contains SOCI core and the C++ implementation of PostgreSQL.'
+      doc_source_url:  http://soci.sourceforge.net/doc/release/4.0/
+
+about:
+  home: http://soci.sourceforge.net/
+  license: BSD-3-Clause
+  license_family: BSD
+  license_file: LICENSE
+  summary: 'SOCI is a general purpose database written in C++.'
```

This is quite misleading, SOCI isn't a database at all. I'd suggest:
```suggestion
  summary: 'SOCI is a general purpose database access library written in C++.'
```

The summaries for the individual libraries above are also rather strangely worded, they should say "implementation of Foo backend" rather than "C++ implementation of Foo".

---

Hum! Good point, will change thanks @vads